README.md

```diff
@@ -843,6 +843,7 @@ Table of Contents
 ## Forms
   * [FabForm](https://fabform.io/) - Form backend platform for smart developers. Free plan allows 250 form submissions per month. Friendly modern GUI. Integrates with Google Sheets, Airtable, Slack, Email and others.
   * [99inbound.com](https://www.99inbound.com/) - Build forms and share them online. Get an email or Slack message for each submission. Free plan has 2 forms, 100 entries per month, basic email & Slack.
+  * [Crove.app](https://crove.app/) - Design mobile-friendly multi-step forms with branching logic. Connect with other apps using Zapier and Webhooks. Free for 3 documents per month.
```

Comment:
So you can fill in a form 3 times a month for free, am I reading it right? or 3 documents filled in often?

```
What is a generated document?

A document generated from template either by automation or by filling the form is called as a created document. Let’s say if you create a contract by filling a questionnaire in the form and click on form submit button to generate document then it is considered “one” generated document.
```